### ISSUE TITLE
fail to fire up container

### ISSUE THREAD

**Issue by eddyizm** [photoprism/photoprism#677]
I was able to easily fire up and set this up last week without issue. Today, I was working on making sure the docker container restarted on system reboot as it was not doing so. It turned out docker was not enabled as a service at this point. 

So I enabled it  : `sudo systemctl enable docker`

Dumped a few more photos into the directory and went to rescan. I found all the images were gone, replaced with the broken black folder icon across everything. Disabled docker, reboot the machine and now when I tried to run docker compose, it errors out, tried with sudo to no avail. 

Question is, did I do something to blow it out or should I look elsewhere? I don't think I changed anything in the working yml file except the restart line: `restart: always`

Any help appreciated, this is an amazing app! 
```
sudo docker-compose up -d
Traceback (most recent call last):
  File "urllib3/connectionpool.py", line 677, in urlopen
  File "urllib3/connectionpool.py", line 392, in _make_request
  File "http/client.py", line 1252, in request
  File "http/client.py", line 1298, in _send_request
  File "http/client.py", line 1247, in endheaders
  File "http/client.py", line 1026, in _send_output
  File "http/client.py", line 966, in send
  File "docker/transport/unixconn.py", line 43, in connect
ConnectionRefusedError: [Errno 111] Connection refused

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "requests/adapters.py", line 449, in send
  File "urllib3/connectionpool.py", line 727, in urlopen
  File "urllib3/util/retry.py", line 403, in increment
  File "urllib3/packages/six.py", line 734, in reraise
  File "urllib3/connectionpool.py", line 677, in urlopen
  File "urllib3/connectionpool.py", line 392, in _make_request
  File "http/client.py", line 1252, in request
  File "http/client.py", line 1298, in _send_request
  File "http/client.py", line 1247, in endheaders
  File "http/client.py", line 1026, in _send_output
  File "http/client.py", line 966, in send
  File "docker/transport/unixconn.py", line 43, in connect
urllib3.exceptions.ProtocolError: ('Connection aborted.', ConnectionRefusedError(111, 'Connection refused'))

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "docker/api/client.py", line 205, in _retrieve_server_version
  File "docker/api/daemon.py", line 181, in version
  File "docker/utils/decorators.py", line 46, in inner
  File "docker/api/client.py", line 228, in _get
  File "requests/sessions.py", line 543, in get
  File "requests/sessions.py", line 530, in request
  File "requests/sessions.py", line 643, in send
  File "requests/adapters.py", line 498, in send
requests.exceptions.ConnectionError: ('Connection aborted.', ConnectionRefusedError(111, 'Connection refused'))

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "bin/docker-compose", line 3, in <module>
  File "compose/cli/main.py", line 67, in main
  File "compose/cli/main.py", line 123, in perform_command
  File "compose/cli/command.py", line 69, in project_from_options
  File "compose/cli/command.py", line 132, in get_project
  File "compose/cli/docker_client.py", line 43, in get_client
  File "compose/cli/docker_client.py", line 170, in docker_client
  File "docker/api/client.py", line 188, in __init__
  File "docker/api/client.py", line 213, in _retrieve_server_version
docker.errors.DockerException: Error while fetching server API version: ('Connection aborted.', ConnectionRefusedError(111, 'Connection refused'))
[4696] Failed to execute script docker-compose
```

**Comment by lastzero** [comment]
These errors don't seem to be related to PhotoPrism. Our demo is working, so it's not a general issue with our Docker image.

**Comment by eddyizm** [comment]
@lastzero fair enough. I don't know how my system went sideways as everything was fine but I guess this is what happens when you tinker. Thanks for the response.

**Comment by eddyizm** [comment]
I am not sure if this is the exact sequence of events but my solution was to add my user to the docker group. 
`sudo usermod -aG docker $USER`

Perhaps there was no group as I have just started using docker and it was not running as a service until I enabled it. This allowed the container access to the daemon - at least that's what I gathered. Container fired back up and now I'm reindexing all my images. All the albums and such are preserved and seem to be loading.
